apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V61__set_bene_id_numeric_constraint.sql

```diff
@@ -0,0 +1,4 @@
+-- add a constraint using the indexed column bene_id_numeric;
+-- psql concurrency enabled which treats this as non-transactional
+CREATE UNIQUE INDEX ${logic.index-create-concurrently} beneficiaries_bene_id_numeric_idx
+    ON public.beneficiaries (bene_id_numeric);
```

Comment:
I thought we were going to forgo the concurrent index build in favor of a single script since this takes only a few minutes and simplifies the migration. Did something come up in testing that changed that?

Reply:
Yup, will remove the concurrency; building index will lock out writes, but not reads.